src/text-editor.coffee

```diff
@@ -2984,3 +3015,9 @@ class TextEditor extends Model
     "<TextEditor #{@id}>"
 
   logScreenLines: (start, end) -> @displayBuffer.logLines(start, end)
+
+
+module.exports = {
+  TextEditor,
+  LineNumberGutterName,
```

Comment:
I'm okay not defining a constant here for the 'line-number' gutter name and just standardizing on a specific string. Since we're not in a typed environment I'm not sure it adds enough to warrant a "constant". Also, I'd prefer to keep exporting the entire class rather than breaking out individual fields like this.